src/registration/pointsetentropyregistrar.py

```diff
@@ -0,0 +1,100 @@
+# Filename:     MeshToMeshElastix
+# Contributors: Tom Birdsong
+# Purpose:  Python implementation of mesh-to-mesh registration
+#     using point set entropy-based registration
+# Referenced https://github.com/InsightSoftwareConsortium/ITKTrimmedPointSetRegistration/blob/master/examples/TrimmedPointSetRegistrationExample.cxx
+#     for development purposes
+
+#!/usr/bin/env python3
+
+import logging
+import os
+import pytest
+
+import itk
+from .meshtomeshregistrar import MeshToMeshRegistrar
+
+class PointSetEntropyRegistrar(MeshToMeshRegistrar):
+    # Common definitions
+    MAX_ITERATIONS = 200
+
+    Dimension = 3
+    PixelType = itk.F
+    ImageType = itk.Image[PixelType, Dimension]
+    FixedImageType = ImageType
+    MovingImageType = ImageType
+
+    MaskPixelType = itk.UC
+    MaskImageType = itk.Image[MaskPixelType, Dimension]
+    FixedPointSetType = itk.PointSet[itk.F, Dimension]
+
+    MeshType = itk.Mesh[itk.F, Dimension]
+
+    # Class-specific definitions
+    PointSetType = itk.PointSet[itk.UI,Dimension]
+    MetricType = itk.JensenHavrdaCharvatTsallisPointSetToPointSetMetricv4[PointSetType]
+    shiftScalesType = \
+      itk.RegistrationParameterScalesFromPhysicalShift[MetricType]
+    TransformType = itk.AffineTransform[itk.D,Dimension]
+    OptimizerType = itk.RegularStepGradientDescentOptimizerv4
+    RegistrationType = itk.ImageRegistrationMethodv4[ImageType,ImageType]#,TransformType, ImageType, PointSetType ]
+    TransformFilterType = itk.TransformMeshFilter[MeshType,MeshType,itk.Transform[itk.F,Dimension,Dimension]]
+
+    def __init__(self):
+      pass
+  
+    def register(self, fixedMesh:MeshType, movingMesh:MeshType,
+                  outFile:str=None, verbose=False) -> MeshType:
+      fixedPointSet = self.PointSetType.New(Points=fixedMesh.GetPoints())
+      movingPointSet = self.PointSetType.New(Points=movingMesh.GetPoints())
```

Comment:
We should use snake_case for idiomatic Python instance variable naming, `movingPointSet` -> `moving_point_set`.